CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 ### Added
+- `ContainerFuture` returns error logs as an error message when a job
```

Comment:
I'm a bit confused by this changelog message. It seems like it's describing the shortcoming this PR is intended to address rather than the change. Is that correct? I think it'd be better to use something along the lines of the PR title ("ContainerFuture propagates error from logs") for this changelog entry.